tests/components/application-connector/test/application-gateway/simple_test.go

```diff
@@ -1,36 +0,0 @@
-package application_gateway_test
-
-import (
-	"context"
-	"io/ioutil"
-	"net/http"
-
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-)
-
-func (gs *GatewaySuite) TestSimpleCases() {
-	app, err := gs.cli.ApplicationconnectorV1alpha1().Applications().Get(context.Background(), "test-app", v1.GetOptions{})
-
-	gs.Nil(err)
-
-	for _, service := range app.Spec.Services {
-		gs.Run(service.DisplayName, func() {
-			for _, entry := range service.Entries {
-				if entry.Type != "API" {
-					gs.T().Log("Skipping event entry")
-					continue
-				}
-				gs.T().Log("Calling", entry.CentralGatewayUrl)
-				res, err := http.Get(entry.CentralGatewayUrl)
-				gs.Nil(err)
-				if err == nil {
-					body, err := ioutil.ReadAll(res.Body)
```

Comment:
Body logging was very useful for debugging. It showed errors returned from the gateway. I don't think we should be removing it

Reply:
No problem :D but can you follow this suggestion -> https://github.com/kyma-project/kyma/pull/14791#discussion_r919926899